src/components/Tabs/index.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react'
+import {Tab, Tabs, TabList, TabPanel} from 'react-tabs'
```

Comment:
Ok we figured out it was already in the package.json before this PR, which is why it was working and why it didn't show up in the diff.